docs/theme.yml

```diff
@@ -32,13 +32,17 @@ perms:
     foreground: Red
   other_execute:
     foreground: Green
+
 filenames:
   # Just change the icon glyph
   Cargo.toml: {icon: {glyph: 🦀}}
   Cargo.lock: {icon: {glyph: 🦀}}
+
 extensions:
   # Change the filename color and icon
+  # NOTE: not all unicode glyphs support color changes
   rs: {filename: {foreground: Red}, icon: {glyph: 🦀}}
+
   # Change the icon glyph and color
   nix: {icon: {glyph: ❄, style: {foreground: White}}}
 icon_overrides:
```

Comment:
The whole part from `icon_overrides:` to EOF is actually just junk that I somehow missed when refactoring the theme file PR and thus can be removed.

Reply:
likely was my fault during the rebase